### ISSUE TITLE
Distributed freeze takes a long time and causes a quorum not reached error

### ISSUE THREAD

**Issue by wouterv** [orientechnologies/orientdb#4950]
I'm trying to setup backups for distributed orient. We are using 2.0.15. We have 3 nodes, one have an lvm partition. When calling freeze however we have two problems: 
- freezing takes a long time (15 seconds)
- the server logs throws quorum 2 not reached errors, which is weird as the other 2 nodes are still up.

The exception on the server were we do the freeze:

```
2015-09-10 09:00:48:267 INFO  Freezing database 'plocal:/opt/orientdb/databases/beeyond' [ONetworkProtocolBinary]
2015-09-10 09:01:15:309 INFO  Realising database 'plocal:/opt/orientdb/databases/beeyond' [ONetworkProtocolBinary]
2015-09-10 09:01:20:279 WARNING [node1441116357316] timeout (9001ms) on waiting for synchronous responses from nodes=[node1441116357316, node1441113490099, node1441113441882] responsesSoFar=[node1441113490099, node1441113441882] request=id=151558 from=node1441116357316 task=tx userName=admin [OHazelcastDistributedDatabase]
2015-09-10 09:01:20:280 WARNING [node1441116357316] no response received from local node about request id=151558 from=node1441116357316 task=tx userName=admin [ODistributedResponseManager]
2015-09-10 09:01:20:280 WARNING [node1441116357316] detected 1 node(s) in timeout or in conflict and quorum (2) has not been reached, rolling back changes for request (id=151558 from=node1441116357316 task=tx userName=admin) [ODistributedResponseManager]
2015-09-10 09:01:20:280 WARNING [node1441116357316] Quorum 2 not reached for request (id=151558 from=node1441116357316 task=tx userName=admin). Elapsed=9006ms No server in conflict. Received: {node1441116357316=waiting-for-response, node1441113490099=[], node1441113441882=[]} [ODistributedResponseManager]
2015-09-10 09:01:20:281 WARNING [node1441116357316] sending undo message for request (id=151558 from=node1441116357316 task=tx userName=admin) to server node1441113490099 [ODistributedResponseManager]
2015-09-10 09:01:20:285 WARNING [node1441116357316] sending undo message for request (id=151558 from=node1441116357316 task=tx userName=admin) to server node1441113441882 [ODistributedResponseManager]Sent run-time exception to the client /172.17.192.3:58172: com.orientechnologies.orient.server.distributed.ODistributedException: Error on executing distributed request (id=151558 from=node1441116357316 task=tx userName=admin) against database 'beeyond.[]' to nodes [node1441116357316, node1441113490099, node1441113441882]
com.orientechnologies.orient.server.distributed.ODistributedException: Error on executing distributed request (id=151558 from=node1441116357316 task=tx userName=admin) against database 'beeyond.[]' to nodes [node1441116357316, node1441113490099, node1441113441882]
    at com.orientechnologies.orient.server.hazelcast.OHazelcastDistributedDatabase.send2Nodes(OHazelcastDistributedDatabase.java:195)
    at com.orientechnologies.orient.server.hazelcast.OHazelcastPlugin.sendRequest(OHazelcastPlugin.java:346)
    at com.orientechnologies.orient.server.distributed.ODistributedStorage.commit(ODistributedStorage.java:945)
    at com.orientechnologies.orient.core.tx.OTransactionOptimistic.doCommit(OTransactionOptimistic.java:488)
    at com.orientechnologies.orient.core.tx.OTransactionOptimistic.commit(OTransactionOptimistic.java:147)
    at com.orientechnologies.orient.core.db.document.ODatabaseDocumentTx.commit(ODatabaseDocumentTx.java:2437)
    at com.orientechnologies.orient.core.db.document.ODatabaseDocumentTx.commit(ODatabaseDocumentTx.java:2407)
    at com.orientechnologies.orient.server.network.protocol.binary.ONetworkProtocolBinary.commit(ONetworkProtocolBinary.java:1092)
    at com.orientechnologies.orient.server.network.protocol.binary.ONetworkProtocolBinary.executeRequest(ONetworkProtocolBinary.java:394)
    at com.orientechnologies.orient.server.network.protocol.binary.OBinaryNetworkProtocolAbstract.execute(OBinaryNetworkProtocolAbstract.java:217)
    at com.orientechnologies.common.thread.OSoftThread.run(OSoftThread.java:69)
Caused by: com.orientechnologies.orient.server.distributed.ODistributedException: Quorum 2 not reached for request (id=151558 from=node1441116357316 task=tx userName=admin). Elapsed=9006ms No server in conflict. Received: {node1441116357316=waiting-for-response, node1441113490099=[], node1441113441882=[]}
    at com.orientechnologies.orient.server.distributed.ODistributedResponseManager.manageConflicts(ODistributedResponseManager.java:585)
    at com.orientechnologies.orient.server.distributed.ODistributedResponseManager.getFinalResponse(ODistributedResponseManager.java:349)
    at com.orientechnologies.orient.server.hazelcast.OHazelcastDistributedDatabase.waitForResponse(OHazelcastDistributedDatabase.java:358)
    at com.orientechnologies.orient.server.hazelcast.OHazelcastDistributedDatabase.send2Nodes(OHazelcastDistributedDatabase.java:192)
    ... 10 more
```

Error logs in other server:

```
2015-09-10 09:01:12:986 WARNING [node1441113490099] timeout (9000ms) on waiting for synchronous responses from nodes=[node1441116357316, node1441113490099, node1441113441882] responsesSoFar=[node1441113490099, node1441113441882] request=id=151556 from=node1441113490099 task=tx userName=admin [OHazelcastDistributedDatabase]
```

And the other server:

```
2015-09-10 09:01:16:587 WARNING [node1441113441882] timeout (9001ms) on waiting for synchronous responses from nodes=[node1441116357316, node1441113490099, node1441113441882] responsesSoFar=[node1441113490099, node1441113441882] request=id=151557 from=node1441113441882 task=tx userName=admin [OHazelcastDistributedDatabase]
```

**Comment by wouterv** [comment]
I found in a non-distributed database where there was a lot less data, but a lot more clusters that freeze can take even longer. Time was at least a minute.

**Comment by gauravarora** [comment]
We're facing a similar problem. When can we expect this to be closed?

**Comment by wouterv** [comment]
The freeze speed is unchanged, but the quorum issues were resolved in 2.1.7

**Comment by wouterv** [comment]
Also we added this to the server config to work around the timeouts:

<entry value="30000" name="distributed.crudTaskTimeout"/>
